<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="but_openFile">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>open project</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_path1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="but_name">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>390</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_email">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>commit history</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>diff</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_commit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>commit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_init">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Init</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_restore">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Restore</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_clone">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_managePeers">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Manage Peers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_delete">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListView" name="list_commit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>481</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="but_diff">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>diff</string>
    </property>
   </widget>
   <widget class="QListView" name="list_diff">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>311</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpne_Project"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFIle"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpne_Project">
   <property name="text">
    <string>Opne Project</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
